api/mmlogic.proto

```diff
@@ -54,24 +54,23 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
   // https://github.com/grpc-ecosystem/grpc-gateway/blob/master/examples/proto/examplepb/a_bit_of_everything.proto
 };
 
-message RetrievePoolRequest {
+message GetPoolTicketsRequest {
   // The Pool representing the set of Filters to be queried.
   Pool pool = 1;
 }
 
-message RetrievePoolResponse {
+message GetPoolTicketsResponse {
   // The Tickets that meet the Filter criteria requested by the Pool.
   repeated Ticket ticket = 1;
 }
 
 // The MMLogic API provides utility functions for common MMF functionality such
 // as retreiving Tickets from state storage.
 service MmLogic {
-  // RetrievePool gets the list of Tickets that match every Filter in the
+  // GetPoolTickets gets the list of Tickets that match every Filter in the
```

Comment:
This is not a meaningful API name. I wonder if this query is too generic or so.

Reply:
how about GetFilteredTickets?

Reply:
Can you run this by @sawagh? The names should be consistent and map to HTTP conventions.